content-src/experiments/cliqz.yaml

```diff
@@ -96,5 +96,5 @@ contributors:
     display_name: 'Cliqz GmbH'
     avatar: /static/images/experiments/cliqz/avatars/cliqz-avatar.jpg
 installation_count: 1
-created: '2017-01-11T00:06:00.00Z'
+created: '2016-12-01T00:06:00.00Z'
```

Comment:
@ckprice i set this to an inoffensive date for people who have old versions of the add-on